Mathlib/Algebra/Module/LinearMap/Pointwise.lean

```diff
@@ -20,8 +20,8 @@ variable [AddCommMonoid M] [AddCommMonoid M₁] [AddCommMonoid M₂] [AddCommMon
 variable [Module R M] [Module R M₁] [Module R M₂] [Module S N]
 variable {F : Type*} (h : F)
 
-@[simp]
-theorem image_smul_setₛₗ [SemilinearMapClass F σ M N] (c : R) (s : Set M) :
+-- @[simp] Not longer applies to itself #8386
```

Comment:
```suggestion
-- @[simp] No longer applies to itself #8386
```
I'm not sure I understand this comment. Can you clarify? If you mean that sigma can't be found by whatever so that the simp lemma doesn't fire, it's maybe worth registering as a simp lemma the special case where sigma=id (`image_smul_set` below)?

Reply:
I've added a more extensive comment and added `image_smul_set` as a `@[simp]` lemma.